### ISSUE TITLE
Export as Markdown & Export as HTML does not export nested documents but only the root document

### ISSUE THREAD

**Issue by Mag1cByt3s** [outline/outline#6884]
When you click on Download=>Markdown or Download=>HTML it will only export the top / root page but not the whole nested document structure into a zip file. Instead it only generates one html / md file. This feature was working flawlessly on older versions.

**To Reproduce**
1. Go to any document
2. Click on 'Download => HTML' or 'Download => Markdown'
3. Get only one file downloaded (the root file you selected)

**Expected behavior**
You should get a zip file containing all the files from that document in a nested manner.

**Screenshots**
Not applicable as it does not help to describe the issue.

**Outline (please complete the following information):**
- Install: self hosted
- Version: 4cb0f8bb94275f7ebf7f9056d6ca2a485dcc961f

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser Thorium (Chromium) 
 - Version 123.0.6312.133

**Comment by tommoor** [comment]
Removed bug label as this is expected behavior. It might be nice to have an option to export nested documents though

**Comment by matbgn** [comment]
That would be really nice! It could also be a nice workaround for exporting large collections that hit the 2GB download limit with assets.

**Comment by matbgn** [comment]
Still nice2have feature

**Comment by coneillpj** [comment]
+1 for this

**Comment by msidqi** [comment]
is it possible to download nested documents from api ?

**Comment by coneillpj** [comment]
> is it possible to download nested documents from api ?

yes but it's up to you to write the logic for finding the nested docs via multiple api queries. So you're basically exporting one doc at a time
